enterprise/core-edge/src/main/java/org/neo4j/coreedge/server/EnterpriseCoreEdgeFacadeFactory.java

```diff
@@ -36,7 +35,7 @@
     public GraphDatabaseFacade newFacade( File storeDir, Map<String, String> params, Dependencies dependencies,
                                           GraphDatabaseFacade graphDatabaseFacade )
     {
-        params.put( Configuration.editionName.name(), "Enterprise" );
+        params.put( Configuration.editionName.name(), "Core-Edge Enterprise" );
```

Comment:
As I've said earlier, we should probably consider making edition name an enum. There is already one in UDC called `Edition`. Maybe it is just ok to move it to kernel.